doczrc.js

```diff
@@ -50,15 +50,34 @@ const projectPlugin = () =>
     },
   });
 
+/**
+ * Atlantis contains some private components. These are indicated by an
+ * empty `.private` file in the component's directory.
+ */
 function privateComponentReadmies() {
```

Comment:
Speeeling... Either this
```suggestion
function privateComponentReadmes() {
```
or something like
```suggestion
function privateComponentVisibility() {
```